Week7/app/src/main/java/com/kulloveth/covid19virustracker/data/db/StatusDao.kt

```diff
@@ -0,0 +1,20 @@
+package com.kulloveth.covid19virustracker.data.db
+
+import androidx.paging.DataSource
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.kulloveth.covid19virustracker.model.CountryStatus
+
+@Dao
+interface StatusDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
I like how you've structured this Insert query and used the REPLACE conflict strategy.